packages/shared/src/lib/timezones.ts

```diff
@@ -587,3 +588,30 @@ export const getTimeZoneIcon = (timezone: string): FC<IconProps> => {
 
   return hour >= 6 && hour < 18 ? DaytimeIcon : NighttimeIcon;
 };
+
+export const DEFAULT_TIMEZONE = 'Etc/UTC';
+
+export const dateFormatInTimezone = (
+  date: Date,
+  format: string,
+  timezone?: string,
+): string => {
+  return formatInTimeZone(date, timezone || DEFAULT_TIMEZONE, format);
+};
+
+export const isSameDayInTimezone = (
+  date1: Date,
+  date2: Date,
+  timezone?: string,
+): boolean => {
+  const dateFormat = 'dd-MM-yyyy';
+
+  if (globalThis.isNaN(date1.getTime()) || globalThis.isNaN(date2.getTime())) {
```

Comment:
Why don't use `Number.isNaN()` directly? This should work both on client/server ✔️

Reply:
Ah right, initially I was checking the `Date` directly which does not work with `Number.isNaN`, but moved to `getTime` later. Updated.